dask/dataframe/io/json.py

```diff
@@ -209,6 +212,9 @@ def read_json(
     if path_converter is None:
         path_converter = lambda x: x
 
+    # Handle engine string (Pandas>=2.0)
```

Comment:
It's probably worth raising an informative error if `engine=` is a string and an older version of `pandas` is being used